README.md

```diff
@@ -124,9 +130,21 @@ For more information, see the RIOT website:
 https://www.riot-os.org
 
 
+[api-badge]: https://img.shields.io/badge/docs-API-informational.svg
+[api-link]: http://riot-os.org/api/
+[irc-badge]: https://img.shields.io/badge/chat-IRC-brightgreen.svg
+[irc-link]: https://webchat.freenode.net?channels=%23riot-os
+[license-badge]: https://img.shields.io/github/license/RIOT-OS/RIOT
+[license-link]: https://github.com/RIOT-OS/RIOT/blob/master/LICENSE
 [master-ci-badge]: https://ci.riot-os.org/RIOT-OS/RIOT/master/latest/badge.svg
 [master-ci-link]: https://ci.riot-os.org/nightlies.html#master
-[irc-badge]: https://img.shields.io/badge/IRC-join%20chat%20%E2%86%92-blue.svg
-[irc-link]: https://webchat.freenode.net?channels=%23riot-os
-[matrix-badge]: https://img.shields.io/badge/Matrix-join%20chat%20%E2%86%92-blue.svg
+[matrix-badge]: https://img.shields.io/badge/chat-Matrix-brightgreen.svg
 [matrix-link]: https://matrix.to/#/#riot-os:matrix.org
+[release-badge]: https://img.shields.io/github/release/RIOT-OS/RIOT.svg
+[release-link]: https://github.com/RIOT-OS/RIOT/releases/latest
+[stackoverflow-badge]: https://img.shields.io/stackexchange/stackoverflow.com/t/%5Briot-os%5D?label=stackoverflow%20questions
+[stackoverflow-link]:  https://stackoverflow.com/questions/tagged/riot-os
+[twitter-badge]: https://img.shields.io/badge/social-Twitter-informational.svg
+[twitter-link]: https://twitter.com/RIOT_OS
+[wiki-badge]: https://img.shields.io/badge/docs-Wiki-informational.svg
+[wiki-link]: http://riot-os.org/api/
```

Comment:
This is not pointing to the wiki. Should we add a badge for the wiki BTW?

Reply:
Whoops!

I think a link to the wiki makes sense: 1) docs seems to be either in the API docs or on the wiki, and 2) the wiki isn't *that* discoverable on GitHub.

Reply:
Nice one!, I would really like to remove the wiki from that list but there is still a lot of info that is only on the wiki...so makes sense to keep it...at least for now, can you fix the link @silkeh?

Reply:
Fixed!